synced-files/.github/workflows/no-response.yml

```diff
@@ -0,0 +1,22 @@
+# NOTE: This is a common file that is overwritten by realm/ci-actions sync service
+# and should only be modified in that repository.
+
+name: No Response
+
+# Both `issue_comment` and `scheduled` event types are required for this Action
+# to work properly.
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    # Schedule for five minutes after the hour, every hour
+    - cron: '5 * * * *'
```

Comment:
BTW, we should only need to run this once a day? @nirinchev

Reply:
Yeah, doesn't matter too much how often we do it - the action itself is super cheap but we can tweak the schedule to once a day.